### PR TITLE
Add a PHP example for Changing Operations in the OpenAPI Documentation

### PR DESCRIPTION
**Pull request by elkuku**
I'd also like to document how to define custom return types - is it even possible?...

### CODE REVIEW

File: core/swagger.md [hunk 1]
```diff
@@ -369,6 +369,54 @@ resources:
...
+ *             "swagger_context"=RABBIT:API_POST
+ *          }
+ *      },
+ * )
+ */
+class Rabbit
+{
+    public const API_POST
+        = [
+            'summary'     => 'Random rabbit picture',
```
Comment (dunglas) [inline core/swagger.md:412]: We usually don't align symbols.

File: core/swagger.md [hunk 2]
```diff
@@ -369,6 +369,54 @@ resources:
...
+            'description' => '# Pop a great rabbit picture by color!'
```
Comment (dunglas) [inline core/swagger.md:413]: You can use the HEREDOC syntax for more clariy:
```suggestion
            'description' => <<<'MD'
# Pop a great rabbit picture by color!

![A great rabbit](https://rabbit.org/graphics/fun/netbunnies/jellybean1-brennan1.jpg
MD,
```

File: core/swagger.md (hunk 1 above)
Comment (elkuku) [reply core/swagger.md]: Fixed :wink:

File: core/swagger.md (hunk 2 above)
Comment (elkuku) [reply core/swagger.md]: Sure no problem, I think it's a matter of taste ;)

**Comment by elkuku** [comment]
I wonder how to integrate documentation for custom response objects. I found https://swagger.io/specification/#response-object but I am unsure how to implement it...
Could be part of this PR?

**Comment by elkuku** [comment]
OK so I found out how to declare return types (including the schema..uff) and also how to show the small padlock icon, so I thought I'd include my findings in this PR ;)

I'd really love to see more documentation for this great product :)

**Comment by elkuku** [comment]
I guess this is ready for merging now :wink:

**Comment by soyuka** [comment]
I 'm not sure to get what added value this has as we already have an example right?

**Comment by elkuku** [comment]
not wanted :(

**Pull request closed without merging**